system/coverage/browser/CodeBrowser.cfc

```diff
@@ -48,15 +48,16 @@ component accessors=true {
 			directoryCreate( browserOutputDir );
 			fileCopy( "#variables.ASSETS_DIR#/js/syntaxhighlighter.js", browserOutputDir );
 			fileCopy( "#variables.ASSETS_DIR#/js/jquery-3.3.1.min.js", browserOutputDir );
+			fileCopy( "#variables.ASSETS_DIR#/js/stupidtable.min.js", browserOutputDir );
 			fileCopy( "#variables.ASSETS_DIR#/css/syntaxhighlighter.css", browserOutputDir );
 			fileCopy( "#variables.ASSETS_DIR#/css/bootstrap.min.css", browserOutputDir );
 		}
 
 		// Create index
-		savecontent variable="local.index" {
+		savecontent variable="index" {
```

Comment:
please change this back to `local`, if not it will be placed as `variables`

Reply:
local is something I added because I like being specific when scoping, but I removed for consistency since nothing else had it.

Reply:
changed it back to local.